--- /dev/null
+++ cookie-factory-0.3.2/meson.build
@@ -0,0 +1,18 @@
+project('cookie-factory-rs', 'rust',
+  version : '0.3.2',
+  meson_version : '>= 1.1.0',
+  default_options : ['rust_std=2018'],
+)
+
+features = []
+features += ['--cfg', 'feature="default"']
+features += ['--cfg', 'feature="std"']
+
+cookie_factory = static_library('cookie_factory', 'src/lib.rs',
+  rust_args : features,
+  rust_crate_type : 'rlib',
+  pic : true,
+)
+
+cookie_factory_dep = declare_dependency(link_with : cookie_factory)
+meson.override_dependency('cookie-factory-rs', cookie_factory_dep)
